javascript/config.yaml

```
@@ -27,9 +27,18 @@ language:
         - npm install
       command: pm2-runtime start app.js -i $(nproc)
     deno:
-      command: deno run --allow-net --allow-read --allow-env app.ts
+      deps:
+        - npm
+      bootstrap:
+        - npm --location=global install pm2
+      command: pm2 start app.ts --interpreter="deno" --interpreter-args="run --allow-net --allow-read --allow-env" -i max &
     bun:
-      command: bun run app.ts
+      deps:
+        - npm
+      bootstrap:
+        - npm --location=global install pm2
```

```suggestion
        - npm -g install pm2
```
not sure why this command does not work